students/JeffBennett/Lesson03/strformat_lab.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+# Task One
+print('\nTask One:\n')
+print('file_{0:0=3}:  {1:5.2f}, {2:.2e}, {3:.2e}'.format
+      (2, 123.4567, 10000, 12345.67))
+
+# Task Two
+print('\nTask Two:\n')
+data = [2, 123.4567, 10000, 12345.67]
+print(f'file_{data[0]:0=3}:  {data[1]:5.2f}, {data[2]:.2e}, {data[3]:.2e}')
+
+
+# Task Three
+print('\nTask Three:\n')
+
+
+def formatter(*tpl):
+    """build a format string with a variable length argument tuple """
+    format_string = ''
+    for item in tpl:
+        format_string += '{:d},'
```

Comment:
You could use the `*` operator to multiply the format strings and simplify this a bit